### PR TITLE
Replace SQLite3 with Postgres

### PR DESCRIPTION
**Pull request by wesdotcool**
@monagupta 

This allows us to put our project on Heroku
Also add a gem for Windows development'

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -4,7 +4,8 @@ source 'https://rubygems.org'
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '~> 5.0.0', '>= 5.0.0.1'
 # Use sqlite3 as the database for Active Record
```
Comment (monagupta) [inline Gemfile]: delete this line and the next?

File: config/database.yml [hunk 2]
```diff
@@ -5,21 +5,24 @@
...
+  adapter: postgresql
   pool: 5
   timeout: 5000
+  username: db_user
 
 development:
   <<: *default
-  database: db/development.sqlite3
+  database: wes_and_mona_dev
+  password: pixpixpix
```
Comment (monagupta) [inline config/database.yml:16]: Is it okay for the passwords for dev and test to be saved here?

**Comment by monagupta** [comment]
LGTM :whale:

File: config/database.yml (hunk 2 above)
Comment (wesdotcool) [reply config/database.yml:16]: Yes. These databases are only going to be on your local machines and they shouldn't hold any production data in them.

**Pull request merged**